### PR TITLE
Add primeng@2.0.1 w/ npm auto-update via single package.json

### PR DESCRIPTION
**Pull request by sashberd**
Pull request for issue: #10134
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/primefaces/primeng
 * Official website (optional, not the repository): http://www.primefaces.org/primeng/#/
 * NPM package url (optional): https://www.npmjs.com/package/primeng
 * License and its reference: MIT, https://github.com/primefaces/primeng/blob/master/LICENSE.md
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 319
   - Count of stars: 5.8k
   - Count of forks: 2.9k
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week: 212778
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/primeng/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "name": "primeng",
+    "filename": "esm2015/primeng.js",
+    "description": "PrimeNG is a collection of rich UI components for Angular 2",
+    "version": "2.0.1",
```
Comment (MattIPv4) [inline ajax/libs/primeng/package.json]: Please remove the version attr, the bot will populate this when it pulls in versioned files.

File: ajax/libs/primeng/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        "files": [
+          "bundles/*.+(js|map)",
+          "esm2015/**/*",
+          "fesm2015/*.+(js|map)",
+          "fesm5/*.+(js|map)",
+          "resources/**/!(*.scss)"
+        ]
+      }
+    ]
+  }
```
Comment (MattIPv4) [inline ajax/libs/primeng/package.json]: Newline at end of file, please

File: ajax/libs/primeng/package.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{
+    "name": "primeng",
```
Comment (MattIPv4) [inline ajax/libs/primeng/package.json]: Indentation of file is not correct, 2 spaces please.

File: ajax/libs/primeng/package.json (hunk 1 above)
Comment (xtuc) [reply ajax/libs/primeng/package.json]:
```suggestion
```

That's ok with the new bot. I will move it once it lands

**Pull request merged**